packages/graph/src/Graph.ts

```diff
@@ -467,9 +445,9 @@ export class Graph extends SVGZoomWidget {
             .attr("class", "graphEdge")
             .style("opacity", 1e-6)
             .on("click.selectionBag", function (d) {
-                context._selection.click(d, d3Event);
+                context._selection.click(d, event);
```

Comment:
Is this going to work the same way d3Event did?

Reply:
This should not have made it into this PR - good catch.